config/config.go

```diff
@@ -40,6 +42,12 @@ type Email struct {
 	To            []string
 }
 
+// Slack holds the configuration for slack notifier
+type Slack struct {
+	Enabled bool
+	URL     string
```

Comment:
the API key will be contained in the URL, right?

Reply:
yes :+1: in config there is also an example with the url format